plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_core_aws.py

```diff
@@ -193,7 +206,11 @@ def launch_jobs(
                 log.info("NOT deleting the cluster (provider.cache_stopped_nodes=true)")
             else:
                 log.info("Deleted the cluster (provider.cache_stopped_nodes=false)")
-            ray_down(launcher.ray_yaml_path)
+            sdk.teardown_cluster(  # type: ignore[attr-defined]
+                config,
```

Comment:
these all look like cluster config, same as before, can we include workersonly and keep min workers in the config?